content/hi/kubernetes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: कुबेरनेट्स (Kubernetes)
+status: completed
+category: अवधारणा
+exclude_search: true
+---
+
+## यह क्या है
+
+कुबेरनेट्स, जिसे अक्सर K8s के रूप में संक्षिप्त किया जाता है, एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेटर है जो आधुनिक इन्फ्रास्ट्रक्चर पर [कंटेनरीकृत](/hi/container/)अनुप्रयोगों के जीवन-चक्र को स्वचालित करता है। यह एक डेटा सेंटर ऑपरेटिंग सिस्टम की तरह है जो एक [वितरित सिस्टम](/distributed_systems/) में चल रहे एप्लिकेशन को प्रबंधित करता है (जैसे आपके लैपटॉप पर ओएस जो आपके ऐप्स को प्रबंधित करता है)।
+
+कुबेरनेट्स एक [क्लस्टर](/cluster/) में [नोड्स](/nodes/) में कंटेनरों को शेड्यूल करता है। यह कई बुनियादी ढांचे के निर्माण को बंडल करता है, जिसे कभी-कभी "आदिम" के रूप में संदर्भित किया जाता है, जैसे कि एक ऐप का एक उदाहरण, लोड बैलेंसर, लगातार भंडारण, और अन्य एक साथ इस तरह से कि उन्हें अनुप्रयोगों में बनाया जा सकता है।
+
+कुबेरनेट्स स्वचालन और एक्स्टेंसिबिलिटी को सक्षम बनाता है, जिससे उपयोगकर्ता घोषणात्मक रूप से पुनरुत्पादित तरीके से अनुप्रयोगों को तैनात कर सकते हैं। कुबेरनेट्स पारिस्थितिकी तंत्र में सॉफ़्टवेयर उत्पाद और प्रोजेक्ट कुबेरनेट्स [API](/application_programming_interface/) का विस्तार करने के लिए उस स्वचालन और एक्स्टेंसिबिलिटी का लाभ उठाते हैं। यह उन्हें कुबेरनेट्स के ऑटोमेशन का लाभ उठाने और अनुभवी कुबेरनेट्स प्रैक्टिशनर्स के लिए अपने टूल्स को अधिक सुलभ बनाने में सक्षम बनाता है।

```

Comment:
If we are writing automation as स्वचालन
We should also write extensibility as विस्तारशीलता